src/components/social-share/social-share.scss

```
@@ -0,0 +1,7 @@
+.__social-share-container {
```

Any reason we prefix this with ` .__`? Maybe something like this?
```suggestion
.socials-share-container {
```

---

Also, in the design we have a `1px border-bottom`, can you please add that one as well?

---

To avoid style collision.

---

Makes sense. Can you please add the `border` tho? 😄